Bottom: ce221cd78930a979aa658fca78848d28f5908a97
Top:    b0c457ec7c1940a7373bf4d5e54fa688a3a73d8f
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-04-14 15:11:48 +0800

Refresh of virtio-net-add-basic-interrupt

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 9deeeaa..95bdee0 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -1431,8 +1431,8 @@ static int virtnet_set_coalesce(struct net_device *dev,
 	    vi->tx_max_coalesced_frames != ec->tx_max_coalesced_frames) {
 		for (i = 0; i < vi->max_queue_pairs; i++) {
 			vi->vdev->config->set_coalesce(vi->vdev, txq2vq(i),
-						ec->rx_max_coalesced_frames,
-						ec->rx_coalesce_usecs);
+						ec->tx_max_coalesced_frames,
+						ec->tx_coalesce_usecs);
 		}
 		vi->tx_coalesce_usecs = ec->tx_coalesce_usecs;
 		vi->tx_max_coalesced_frames = ec->tx_max_coalesced_frames;
